Online-Messaging-App/src/app/home/logout/logout-form.component.scss

```diff
@@ -0,0 +1,2 @@
+a {
```

Comment:
Does this serve a purpose

Reply:
no already removed it on my end. I don't know why its showing up in the push. I double-checked that its gone now but if its still there (for some goddamned reason) just delete it and it should be fine